asr-python-advanced-wordboosting.ipynb

```diff
@@ -346,6 +347,16 @@
     "We perform positive word boosting, as we did in earlier examples, for word `middle`. Note that we are using the same `config` instance, that we created in the above example for negative word boosting for `little`. So it already has the `SpeechContext` instance for `little`. So, now, we just need to add a `SpeechContext` instance for positive word boosting for `middle`."
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "55605d1c-3a4e-4c05-9ed8-f285dd964ba7",
+   "metadata": {},
+   "source": [
+    "Now let us see how we can combine negative and positive word boosting:  \n",
+    "Let us combine the negative word boosting from above example, with positive word boosting to predict the word `bin` instead of `been`. \n",
```

Comment:
```suggestion
    "Let us combine the negative word boosting from the above example, with positive word boosting, to predict the word `bin` instead of `been`. \n",
```

Reply:
You're right that I forgot to add "the" to Nikhil's original copy. I also forgot about the comma after "example". I'm not sure that either comma is needed in this sentence. Which looks/sounds better to you, "Let us combine negative word boosting with positive word boosting to predict..." or "Let us combine negative word boosting, with positive word boosting, to predict..."